allennlp/training/trainer.py

```diff
@@ -102,6 +103,8 @@ def __init__(self,
         if self._cuda_device >= 0:
             self._model = self._model.cuda(self._cuda_device)
 
+        self._log_interval = 10  # seconds
```

Comment:
is the thinking that we might want to allow people to override this? or should it be a module-level constant?

Reply:
There was no such thinking =).  I don't have strong feelings about where this should go; if you do, I can move it.

Reply:
if we don't intend for users to modify it, I'd rather it be a module-level constant `_LOG_INTERVAL` than an instance variable. if we think they might want to, then leave it as is.